proto/bech32ibc/v1beta1/query.proto

```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
-package osmosis.bech32ibc.v1beta1;
+package bech32ibc.bech32ibc.v1beta1;
```

Comment:
Thanks for fixing these! Can we close the PR on Osmosis now/does proto building here work?